travis/expected_europe.lst

```diff
@@ -47,6 +47,14 @@ ggpf08-Tubuai.gtx
 ggpf10-Moorea.gtx
 ggpf10-Tahiti.gtx
 ggspm06v1.gtx
+icegeoid_2011.gtx
+ISL
+ISN_vel_beta.ct2
```

Comment:
sorting order raised by https://travis-ci.com/OSGeo/proj-datumgrid/builds/138843342. ISN_vel_beta files should be put after ISN93_ files

Reply:
I have edited the file.

Reply:
I uploaded new gtx files: Icegeoid_ISN2004.gtx, Icegeoid_ISN93.gtx and Icegeoid_ISN2016.gtx. I will remove icegeoid_2011.gtx